engine/swarm/services.md

```diff
@@ -80,8 +80,10 @@ flag. For more information, see
 
 When you create a service without specifying any details about the version of
 the image to use, the service uses the latest version available in Docker Hub
-or your registry. You can force the service to use a specific version of
-the image in a few different ways, depending on your desired outcome.
+or your registry, or using Notary if content trust is enabled. See the
+[note about content trust](#image_resolution_with_trust).You can force the
```

Comment:
Addressed the feedback by removing the links to the note, and just having the note at the bottom. I think it's more clear that way. Let me know what you think.